### PR TITLE
Add a method for exporting all of your predictions as CSV

### PR DESCRIPTION
**Pull request by michaelwebb76**
Addresses https://github.com/bellroy/predictionbook/issues/245 and https://github.com/bellroy/predictionbook/issues/7

Settings -> Export my predictions as CSV

### CODE REVIEW

File: app/controllers/api/my_predictions_controller.rb [hunk 1]
```diff
@@ -3,25 +3,15 @@
...
-
-    def prediction_hash_array
-      predictions.map do |prediction|
-        PredictionSerializer.new(prediction).serializable_hash
-      end
-    end
-
-    def predictions
-      PredictionsQuery.new(
+      ps = PredictionsQuery.new(
```
Comment (JackKelly-Bellroy) [inline app/controllers/api/my_predictions_controller.rb:6]:
```suggestion
      q = PredictionsQuery.new(
```

Comment (JackKelly-Bellroy) [reply app/controllers/api/my_predictions_controller.rb:6]: Or `query`

File: app/controllers/predictions_controller.rb [hunk 2]
```diff
@@ -52,6 +54,28 @@ def home
     @show_statistics = false
   end
 
+  def mine
+    ps = PredictionsQuery.new(
```
Comment (JackKelly-Bellroy) [inline app/controllers/predictions_controller.rb:58]:
```suggestion
    q|query = PredictionsQuery.new(
```

File: app/controllers/api/my_predictions_controller.rb (hunk 1 above)
Comment (michaelwebb76) [reply app/controllers/api/my_predictions_controller.rb:6]: This line is initializing an instance of `PredictionsQuery`, invoking `call` on it to return a list of predictions that match the query parameters and then throws the instance away. So I think `q` is misleading.

File: app/controllers/predictions_controller.rb (hunk 2 above)
Comment (michaelwebb76) [reply app/controllers/predictions_controller.rb:58]: As above

File: app/controllers/api/my_predictions_controller.rb (hunk 1 above)
Comment (JackKelly-Bellroy) [reply app/controllers/api/my_predictions_controller.rb:6]: I agree - missed the `.call` at the end; too used to function calls appearing in prefix position.

**Pull request merged**